scripts/commerce.js

```diff
@@ -37,6 +37,7 @@ export const CUSTOMER_LOGIN_PATH = `${CUSTOMER_PATH}/login`;
 export const CUSTOMER_ACCOUNT_PATH = `${CUSTOMER_PATH}/account`;
 export const CUSTOMER_FORGOTPASSWORD_PATH = `${CUSTOMER_PATH}/forgotpassword`;
 export const SALES_ORDER_VIEW_PATH = '/sales/order/view/';
+export const CUSTOMER_REQUISITION_LISTS_PATH = `${CUSTOMER_PATH}/requisition-lists`;
```

Comment:
@svera I believe the link is still wrong, it should be `${CUSTOMER_PATH}/requisition-list-view`to point to the correct document in DA (https://da.live/edit#/adobe-commerce/boilerplate-b2b/customer/requisition-list-view)

I would also consider renaming `CUSTOMER_REQUISITION_LISTS_PATH` to something like `CUSTOMER_REQUISITION_LIST_DETAILS_PATH` maybe?